site/admission/index.php

```diff
@@ -38,51 +39,51 @@
   </div>
   <section class="page-section">
     <div class="admission-info">
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
-        magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut.</p>
+      <?php echo"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
```

Comment:
This isn't really doing much, it's just saying put this text here. Instead I want you to move some of your code to other files and then use the `template(file.php, array())` function to include those other files in this one. Lets chat during studio to make sure we're on the same page. Thanks!